src/features/Chat/components/ActiveWindow/index.tsx

```diff
@@ -26,7 +27,13 @@ import MessageList from './MessageList';
 
 const ActiveWindow = () => {
   const { isTablet } = useGetLayoutMode();
-  const chat = useAppSelector(selectActiveChat);
+  const dispatch = useAppDispatch();
+
+  const activeChat: ChatBuddy | null = useAppSelector(selectActiveChat);
+  const defaultChat: ChatBuddy | null = useAppSelector(selectDefaultChat);
+
+  const chat: ChatBuddy = activeChat ?? defaultChat;
```

Comment:
The return signature of the selector gives already the type. I would remove the explicit type annotation

Reply:
Yes true, fixed this too!